src/sagas/user.saga.ts

```diff
@@ -12,21 +12,43 @@ function* login(action) {
   const { email, password } = action.payload;
 
   try {
-    //const token =
-    yield userService.login(email, password);
-
+    const resData = yield userService.login(email, password);
+    console.log(resData);
+    const tokenExpirationDate = new Date(new Date().getTime() + 1000 * 60 * 60);
```

Comment:
From someone reading this i cannot understand what is the expiration date, would suggest extracting to a variable, also would be better to use a const here for the interval :)